tests/config.ts

```diff
@@ -1,8 +1,10 @@
 // Copyright (C) 2024, Nuklai. All rights reserved.
 // See the file LICENSE for licensing terms.
 
+// API_HOST: 'http://99.81.105.110:9650',
+
 export const TEST_CONFIG = {
-  API_HOST: 'http://127.0.0.1:9650',
+  API_HOST: 'http://99.81.105.110:9650',
```

Comment:
Could we revert this change to using 127.0.0.1?

Reply:
Oops!
That's not supposed to be included in this, will revert